consensus/polybft/mocks_test.go

```diff
@@ -298,6 +300,24 @@ func (t *transportMock) Gossip(message interface{}) {
 	_ = t.Called(message)
 }
 
+var _ checkpointBackend = &checkpointBackendMock{}
```

Comment:
Why do you need to init it? maybe it's another way to check statically.

```go
var _ checkpointBackend = (*checkpointBackendMock)(nil)
```

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/852/commits/a82711ed9b25839a7f003c5f53018db390793148